js/pokemon.js

```diff
@@ -0,0 +1,85 @@
+function getPokemonData(name) {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon?limit=76/');
+  xhr.responseType = 'json';
+
+  xhr.addEventListener('load', function () {
+    const genOne = xhr.response;
+
+    for (let i = 0; i < genOne.results.length; i++) {
+      const xhr2 = new XMLHttpRequest();
+      xhr2.open('GET', `${genOne.results[i].url}`);
+      xhr2.responseType = 'json';
+      xhr2.addEventListener('load', function () {
+        const $rowPokemon = document.querySelector('.row-pokemon');
+        $rowPokemon.appendChild(renderPokemon(xhr2.response));
+      });
+      xhr2.send();
+    }
+  });
+  xhr.send();
+}
+
+getPokemonData('bulbasaur');
```

Comment:
You are not using your `name` parameter at all, so there's no need to have one, and you don't need to pass anything in as an argument.